comp/networkpath/npcollector/npcollectorimpl/npcollector.go

```diff
@@ -211,7 +211,7 @@ func (s *npCollectorImpl) runTracerouteForPath(ptest *pathteststore.PathtestCont
 		DestHostname: ptest.Pathtest.Hostname,
```

Comment:
Based on:

https://github.com/DataDog/datadog-agent/blob/ad414dd63275ea540c38aab8d2ff0cf9a6de773e/pkg/networkpath/traceroute/tcp/tcpv4.go#L106-L110

and 

https://github.com/insomniacslk/dublin-traceroute/blob/master/go/dublintraceroute/probes/probev4/udpv4.go#L196-L197

it seems timeout doesn't have the same meaning in UDP and TCP.

Reply:
We'll change this as part of the other card